ethereum/core/src/main/java/org/hyperledger/besu/ethereum/storage/keyvalue/KeyValueStoragePrefixedKeyBlockchainStorage.java

```diff
@@ -26,64 +33,58 @@
 import org.hyperledger.besu.plugin.services.storage.KeyValueStorage;
 import org.hyperledger.besu.plugin.services.storage.KeyValueStorageTransaction;
 
-import java.nio.charset.StandardCharsets;
 import java.util.Collection;
 import java.util.List;
 import java.util.Optional;
 
-import com.google.common.collect.Lists;
 import org.apache.tuweni.bytes.Bytes;
 import org.apache.tuweni.bytes.Bytes32;
 import org.apache.tuweni.units.bigints.UInt256;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class KeyValueStoragePrefixedKeyBlockchainStorage implements BlockchainStorage {
-
-  private static final Bytes CHAIN_HEAD_KEY =
-      Bytes.wrap("chainHeadHash".getBytes(StandardCharsets.UTF_8));
-  private static final Bytes FORK_HEADS_KEY =
-      Bytes.wrap("forkHeads".getBytes(StandardCharsets.UTF_8));
-  private static final Bytes FINALIZED_BLOCK_HASH_KEY =
-      Bytes.wrap("finalizedBlockHash".getBytes(StandardCharsets.UTF_8));
-  private static final Bytes SAFE_BLOCK_HASH_KEY =
-      Bytes.wrap("safeBlockHash".getBytes(StandardCharsets.UTF_8));
-
+  private static final Logger LOG =
+      LoggerFactory.getLogger(KeyValueStoragePrefixedKeyBlockchainStorage.class);
   private static final Bytes VARIABLES_PREFIX = Bytes.of(1);
```

Comment:
perhaps rename to DEPRECATED_VARIABLES_PREFIX

Reply:
added the `Deprecated` annotation